server/store.go

```diff
@@ -20,126 +21,146 @@ const (
 	StoreReminderKey = "reminder"
 )
 
-func (p *Plugin) storeLastReminderTimeForUser(userID string) error {
-	strTime := strconv.FormatInt(model.GetMillis(), 10)
-	appErr := p.API.KVSet(getReminderKey(userID), []byte(strTime))
-	if appErr != nil {
-		return errors.New(appErr.Error())
-	}
-	return nil
+// OrderElement denotes every element in any of the lists. Contains the item that refers to,
```

Comment:
After reviewing the code some more, I propose 3 concepts/names:
- A `Task` (aka item)
- A `TaskRef` (OrderElement)
- A `List` (aka Order, `[]*TaskRef`)

It may also be worth considering using the term `Issue` to avoid reinventing the wheel and colliding with "agile" naming convention later (Task vs Story???). I still think that `Item` is a bit too abstract.

Reply:
Maybe it is a bit redundant, but if we end up calling the plugin "Reminder" wouldn't it be nice to call the items reminders? May collide with the current remind functionality, but we can work around that.

Reply:
I just realised that if we expect to keep backwards compatibility, using the word reminder will be annoying, since the Reminder key for KV store is "reminder". I will go with Issue then.

Reply:
Personally, `Issue` makes me cringe, but I think it might be the most intention-revealing, generally accepted name for the entity.

Reply:
I have also some problems with issue. Maybe it is because English is not my mother tongue, but whenever I hear Issue I hear Problem, and I do not like transmitting with the todos the feeling of "you have to deal with a problem". Also, I wonder how much it will collide in the mind with other uses of Issue, like GitHub Issues.

Nevertheless, I have no better alternative apart of maybe "Todo" itself, but if we get a new name to the plugin, we may find with it the new name for the element.